app/models/git_repository.rb

```diff
@@ -1,22 +1,23 @@
+require "rugged"
```

Comment:
Not worth holding up the merge, but is there a reason this was added?

Reply:
The ::Rugged::NetworkError caught as a def/rescue in handling_worktree_errors was not handled in isolation when the git_repository_spec was run directly.